addons/project/migrations/11.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def cleanup_translations(cr):
+    """
+    Cleanup translations of adopted templates
+    """
+
+    updated_templates = (
+        'mail_template_data_module_install_project',
+    )
+
+    cr.execute("""
+        SELECT res_id
+        FROM ir_model_data
+        WHERE
+            module = 'project' AND model = 'mail.template' AND name in %s
+    """, (updated_templates,))
+    record_ids = tuple([r[0] for r in cr.fetchall()])
+
+    query = ("""
+        DELETE FROM ir_translation
+        WHERE module = 'calendar' AND res_id IN %s
```

Comment:
calendar?

Reply:
Isn't better to create an openupgradelib function for handling this?

Reply:
I saw that @rruebner already created a PR with this function in openupgradelib (https://github.com/OCA/openupgradelib/pull/87). What should I do, wait until it is merged?

Reply:
Yes, let's merge it and then call it from here.